### PR TITLE
Removed ignoring Space after line break

### PR DESCRIPTION
**Pull request by Snooz82**
Can be merged

### CODE REVIEW

File: atest/robot/parsing/escaping.robot [hunk 1]
```diff
@@ -40,14 +40,7 @@ New Line
     Check Test Case    ${TEST NAME}
 
 Ignoring Space After Newline Is Deprecated
```
Comment (pekkaklarck) [inline atest/robot/parsing/escaping.robot:42]: Outdated name. Should be something like "Space after newline is not ignored".

**Review by pekkaklarck — APPROVED** [review]

**Comment by pekkaklarck** [comment]
Should also remove the note related to handling this from the User Guide.

**Comment by Snooz82** [comment]
Can be merged

**Comment by Snooz82** [comment]
Can be merged!

**Pull request merged**